gplugins/lumerical/tests/test_lumerical_utils.py

```diff
@@ -0,0 +1,249 @@
+from gplugins.lumerical.utils import to_lbr
+from gdsfactory.technology.layer_stack import LayerStack, LayerLevel
+
+
+def test_to_lbr():
```

Comment:
I think a short docstring describing what this does (maybe "Ensure the process file generated from `to_lbr` is importable in MODE, FDTD, and CHARGE") would be useful

Reply:
Addressed: https://github.com/elamdf/gplugins/pull/2/commits/13380939a7263ea27afa7a9943002c2b9d494b32